subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -0,0 +1,145 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[console]]
+== The Gradle Console
+
+[[sec:console_overview]]
+=== Overview
+
+Nearly every Gradle user will experience the command-line interface at some point. Gradle's console output is optimized for rendering performance and usability, showing only relevant information and providing visually appealing feedback.
+
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
:o: we ought to adjust this to not loop via `gifsicle file.gif --no-loopcount > target.gif`